internal/pkg/encoding/json/schema/schema.go

```diff
@@ -120,7 +120,7 @@ func validateDocument(schemaStr []byte, document *orderedmap.OrderedMap) error {
 	return schema.Validate(document.ToMap())
 }
 
-func processErrors(errs []*jsonschema.ValidationError) error {
+func processErrors(errs []*jsonschema.ValidationError, parentIsSchemaErr bool) error {
```

Comment:
Chybu v scheme obajuleme do `SchemaError` typu.
To chceme spravit iba pre najvyssiu/root schema error, nested errors uz nemusime/nechceme obalovat.
To riesi tento novy parameter.